plugins/api-docs/README.md

```diff
@@ -222,6 +222,37 @@ security:
     - [read_pets, write_pets]
 ```
 
+### Override Component Styles
+
+It is possible to override the component styles from theme. To override components of this `api-docs` plugin use the corresponding class key while creating custom theme overrides.
+
+| Component to Override | Class key to use in theme overrides |
+| --------------------- | ----------------------------------- |
+| OpenApiDefinition     | PluginApiDocsOpenApiDefinition      |
+| AysncApiDefinition    | PluginApiDocsAsyncApiDefinition     |
+
+Below is an example of overriding `OpenApiDefinition` component from theme:
+
+```tsx
+export const createCustomThemeOverrides = (
+  theme: BackstageTheme,
+): BackstageOverrides => {
```

Comment:
I would be suprised if `PluginApiDocsOpenApiDefinition` actualy is part of the `BackstageOverrides` type. I don't think this would pass type checks.

Reply:
Also, I don't think this is even needed anymore using the unified theme which is the recommended path when creating a new theme